src/tests/ftest/server/multiengine_persocket.yaml

```diff
@@ -10,40 +10,73 @@ setup:
 server_config:
   name: daos_server
   reboot_waittime: 210
-  engines_per_socket: 2
+  engines_per_socket: 4
```

Comment:
This is incorrect, this should be 2. The config is only using 2 sockets (NUMA 0 & 1) and specifies 2 engines per socket. The title of the PR is also incorrect and should read "2 engines per socket, 4 engines total".

Reply:
When we use "daos_server storage prepare --scm-ns-per-socket 4" to configure 4 engines, we don't have information about 2 engines per socket.
Shouldn't "--scm-ns-per-socket" make clear to customer?

Reply:
scm-ns-per-socket means "how many pmem block devices will be created per NUMA node (socket)", so if you have a 2-socket system (x86_64 norm), you will end up with 4 pmem block devices. you can then create 4 engines on the host. I hope that helps but I don't think I understood your question very well.

Reply:
I think I understand your question better now after reading it again, you want an option for storage prepare to say "configure my storage so that I can create X number of engines". This is nontrivial because it requires analysing the number of NUMA nodes on the system and using that information to divide the sockets into the required number of pmem block devices to satisfy the request. It is possible but I'm not sure if it will increase or decrease the usability, @mjmac any thoughts?

Reply:
ok. "scm-ns-per-socket" should be 2, and total engines in the daos_server.yml config would be be 4, 
I will revert this PR with update.  Thanks.

Reply:
Just to clarify the engine configurations in this PR are correct already, the only parameter that needs updating is "engines_per_socket" which should be set to 2 and not 4. Thanks a lot Ding.